### PR TITLE
Irminger 8 Telemetered Ingestion csvs

### PR DESCRIPTION
**Pull request by cdobs**
Creates the following Irminger 8 telemetered ingestion csvs:
-GI01SUMO-D00008
-GI05MOAS-GL469-D00004
-GI05MOAS-GL537-D00003
-GI05MOAS-PG565-D00002

### CODE REVIEW

File: GI01SUMO/GI01SUMO_D00008_ingest.csv [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+mi.dataset.driver.flort_dj.dcl.flort_dj_dcl_telemetered_driver,/omc_data/whoi/OMC/GI01SUMO/D00008/cg_data/dcl12/flort1/*flort*.log,GI01SUMO-SBD12-02-FLORTD000,telemetered,Available,
+mi.dataset.driver.hyd_o.dcl.hyd_o_dcl_telemetered_driver,/omc_data/whoi/OMC/GI01SUMO/D00008/cg_data/dcl12/hyd2/*hyd*.log,GI01SUMO-SBD12-03-HYDGN0000,telemetered,Available,
+mi.dataset.driver.pco2a_a.sample.pco2a_a_sample_telemetered_driver,/omc_data/whoi/OMC/GI01SUMO/D00008/cg_data/dcl12/pco2a/*pco2a*.log,GI01SUMO-SBD12-04-PCO2AA000,telemetered,Available,
+mi.dataset.driver.wavss_a.dcl.wavss_a_dcl_telemetered_driver,/omc_data/whoi/OMC/GI01SUMO/D00008/cg_data/dcl12/wavss/*wavss*.log,GI01SUMO-SBD12-05-WAVSSA000,telemetered,Available,
+mi.dataset.driver.metbk_a.dcl.metbk_a_dcl_telemetered_driver,/omc_data/whoi/OMC/GI01SUMO/D00008/cg_data/dcl12/metbk2/*metbk*.log,GI01SUMO-SBD12-06-METBKA000,telemetered,Available,
+mi.dataset.driver.fdchp_a.dcl.fdchp_a_dcl_telemetered_driver,/omc_data/whoi/OMC/GI01SUMO/D00008/cg_data/dcl12/fdchp/*fdchp*.log,GI01SUMO-SBD12-08-FDCHPA000,telemetered,Available,
```
Comment (rtravis-whoi) [inline GI01SUMO/GI01SUMO_D00008_ingest.csv:51]: looks good

File: GI05MOAS-GL469/GI05MOAS-GL469_D00004_ingest.csv [hunk 2]
```diff
@@ -0,0 +1,5 @@
+parser,filename_mask,reference_designator,data_source,status,notes
+mi.dataset.driver.moas.gl.engineering.glider_eng_glider_telemetered_driver,/omc_data/whoi/OMC/GI05MOAS-GL469/D00004/merged-from-glider/gi_*.mrg,GI05MOAS-GL469-00-ENG000000,telemetered,Available,
+mi.dataset.driver.moas.gl.flord_m.flord_m_glider_telemetered_driver,/omc_data/whoi/OMC/GI05MOAS-GL469/D00004/merged-from-glider/gi_*.mrg,GI05MOAS-GL469-01-FLORDM000,telemetered,Available,
+mi.dataset.driver.moas.gl.dosta.dosta_abcdjm_glider_telemetered_driver,/omc_data/whoi/OMC/GI05MOAS-GL469/D00004/merged-from-glider/gi_*.mrg,GI05MOAS-GL469-02-DOSTAM000,telemetered,Available,
+mi.dataset.driver.moas.gl.ctdgv.ctdgv_m_glider_telemetered_driver,/omc_data/whoi/OMC/GI05MOAS-GL469/D00004/merged-from-glider/gi_*.mrg,GI05MOAS-GL469-04-CTDGVM000,telemetered,Available,
```
Comment (rtravis-whoi) [inline GI05MOAS-GL469/GI05MOAS-GL469_D00004_ingest.csv:5]: looks good

File: GI05MOAS-GL537/GI05MOAS-GL537_D00003_ingest.csv [hunk 3]
```diff
@@ -0,0 +1,5 @@
+parser,filename_mask,reference_designator,data_source,status,notes
+mi.dataset.driver.moas.gl.engineering.glider_eng_glider_telemetered_driver,/omc_data/whoi/OMC/GI05MOAS-GL537/D00003/merged-from-glider/gi_*.mrg,GI05MOAS-GL537-00-ENG000000,telemetered,Available,
+mi.dataset.driver.moas.gl.flord_m.flord_m_glider_telemetered_driver,/omc_data/whoi/OMC/GI05MOAS-GL537/D00003/merged-from-glider/gi_*.mrg,GI05MOAS-GL537-01-FLORDM000,telemetered,Available,
+mi.dataset.driver.moas.gl.dosta.dosta_abcdjm_glider_telemetered_driver,/omc_data/whoi/OMC/GI05MOAS-GL537/D00003/merged-from-glider/gi_*.mrg,GI05MOAS-GL537-02-DOSTAM000,telemetered,Available,
+mi.dataset.driver.moas.gl.ctdgv.ctdgv_m_glider_telemetered_driver,/omc_data/whoi/OMC/GI05MOAS-GL537/D00003/merged-from-glider/gi_*.mrg,GI05MOAS-GL537-04-CTDGVM000,telemetered,Available,
```
Comment (rtravis-whoi) [inline GI05MOAS-GL537/GI05MOAS-GL537_D00003_ingest.csv:5]: looks good

File: GI05MOAS-PG565/GI05MOAS-PG565_D00002_ingest.csv [hunk 4]
```diff
@@ -0,0 +1,8 @@
...
+mi.dataset.driver.moas.gl.ctdgv.ctdgv_m_glider_telemetered_driver,/omc_data/whoi/OMC/GI05MOAS-PG565/D00002/merged-from-glider/gi_*.mrg,GI05MOAS-PG565-01-CTDGVM000,telemetered,Available,
+mi.dataset.driver.moas.gl.dosta.dosta_abcdjm_glider_telemetered_driver,/omc_data/whoi/OMC/GI05MOAS-PG565/D00002/merged-from-glider/gi_*.mrg,GI05MOAS-PG565-02-DOSTAM000,telemetered,Available,
+mi.dataset.driver.moas.gl.flort_m.flort_m_glider_telemetered_driver,/omc_data/whoi/OMC/GI05MOAS-PG565/D00002/merged-from-glider/gi_*.mrg,GI05MOAS-PG565-03-FLORTM000,telemetered,Available,
+mi.dataset.driver.moas.gl.flort_o.flort_o_glider_telemetered_driver,/omc_data/whoi/OMC/GI05MOAS-PG565/D00002/merged-from-glider/gi_*.mrg,GI05MOAS-PG565-04-FLORTO000,telemetered,Available,
+mi.dataset.driver.nutnr_m.glider.nutnr_m_glider_telemetered_driver,/omc_data/whoi/OMC/GI05MOAS-PG565/D00002/merged-from-glider/gi_*.mrg,GI05MOAS-PG565-05-NUTNRM000,telemetered,Available,
+mi.dataset.driver.moas.gl.parad.parad_m_glider_telemetered_driver,/omc_data/whoi/OMC/GI05MOAS-PG565/D00002/merged-from-glider/gi_*.mrg,GI05MOAS-PG565-06-PARADM000,telemetered,Available,
```
Comment (rtravis-whoi) [inline GI05MOAS-PG565/GI05MOAS-PG565_D00002_ingest.csv:8]: looks good

**Review by rtravis-whoi — APPROVED** [review]

File: GI01SUMO/GI01SUMO_D00008_ingest.csv (hunk 1 above)
Comment (snwhite) [reply GI01SUMO/GI01SUMO_D00008_ingest.csv:51]: Looks good.

File: GI05MOAS-GL469/GI05MOAS-GL469_D00004_ingest.csv (hunk 2 above)
Comment (snwhite) [reply GI05MOAS-GL469/GI05MOAS-GL469_D00004_ingest.csv:5]: Looks good.

File: GI05MOAS-GL537/GI05MOAS-GL537_D00003_ingest.csv (hunk 3 above)
Comment (snwhite) [reply GI05MOAS-GL537/GI05MOAS-GL537_D00003_ingest.csv:5]: Looks good.  But there is not data in the Raw Data Repo.

File: GI05MOAS-PG565/GI05MOAS-PG565_D00002_ingest.csv (hunk 4 above)
Comment (snwhite) [reply GI05MOAS-PG565/GI05MOAS-PG565_D00002_ingest.csv:8]: Looks good.  But there is not data in the Raw Data Repo.

**Review by snwhite — APPROVED** [review]
All files look good, but as noted, not all data is available in the Raw Data Repo.

**Pull request merged**